src/sql/workbench/contrib/objectExplorer/common/serverGroup.contribution.ts

```diff
@@ -52,12 +51,6 @@ const serverTreeConfig: IConfigurationNode = {
 			'type': 'boolean',
 			'default': true,
 			'description': localize('serverTree.useAsyncServerTree', "Use the new async server tree for the Servers view and Connection Dialog with support for new features such as dynamic node filtering. Requires a restart to take effect.")
-		},
-		'serverTree.nodeExpansionTimeout': {
```

Comment:
So we're going back to relying on the providers to complete (either successfully or not) in a reasonable amount of time then? And it'll be expected that if there's an issue with the provider and they never return then node expansion could potentially hang forever?

Reply:
Yes, for now. The timeout are unstable as they are not cleared sometimes causing intermittent issues. We still have fixed race conditions so it is not completely undoing the previous work.